spyder/plugins/completion/providers/languageserver/widgets/serversconfig.py

```diff
@@ -137,8 +137,8 @@ def __init__(self, parent, language=None, cmd='', host='127.0.0.1',
             "port if connecting to an external server, "
             "or to a local one using TCP instead of stdio pipes."
             "<br><br>"
-            "<i>Note</i>: You can use the placeholders <tt>{host}</tt> and "
-            "<tt>{port}</tt> in the server arguments field to automatically "
+            f"<i>Note</i>: You can use the placeholders <tt>{host}</tt> and "
+            f"<tt>{port}</tt> in the server arguments field to automatically "
```

Comment:
This is part of message shown in our interface, so it's not meant to receive values from any variable:

![imagen](https://user-images.githubusercontent.com/365293/165121922-50c5c0c6-746a-431c-98e6-6de81c9317e4.png)